docs/site/content/en/docs/Tutorials/troubleshooting-functions/index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Troubleshooting Functions
+description: Steps to troubleshoot use of Functions in Horreum
+date: 2024-02-27
+weight: 4
+---
+
+## Troubleshooting Functions and Combination Function
+
+Creating Scheam Labels is one of the key capabilities that enables working with loosely defined data. Using a Validation Schema provides useful guarentees data in an uploaded Run JSON conforms to expected data types. Validation is optional and when not used maks the handling of metrics unreliable as data types may change.  
+
+To troubleshoot use of Javascript Functions the following techniques can be used
+
+* The `typeof` function to discover the data-type of the `Extractor` variable, object property or array element in the Label `Combination Function`, using the `Test label calculation` button to reveal the Javascript data-type
+{{% imgproc use-typeof-function Fit "785x769" %}}
+Test label calculation
+{{% /imgproc %}}
+* View the `Labels Log` and display DEBUG log level, reveals syntax errors when the function is executed
+* Use the `Show label values` button, provides a quick visual check of resulting values for the `Run` `Dataset`
+{{% imgproc buttons-on-dataset-view Fit "1091x462 " %}}
+Run Dataset view
+{{% /imgproc %}}
+Effective label values reveal the anticipated value or error text which can be due to the following:
```

Comment:
Can we show a screenshot of `Labels Log` as well, there is debug information displayed showing the input values and the JS evaluation error message.  This is vital for users to debug problems when a function evaluation fails

Reply:
Sure. I will add this.